src/app/signin/signin.component.spec.ts

```diff
@@ -0,0 +1,37 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {SignInComponent} from './signin.component';
+import {AuthService} from '../auth.service';
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import {RouterTestingModule} from '@angular/router/testing';
+import {AngularFireModule} from '@angular/fire';
+import {environment} from '../../environments/environment';
+
+describe('SignInComponent', () => {
+  let component: SignInComponent;
+  let fixture: ComponentFixture<SignInComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        HttpClientTestingModule,
+        RouterTestingModule,
+        AngularFireModule.initializeApp(environment.firebaseConfig),
+      ],
+      declarations: [SignInComponent],
+      providers: [AuthService],
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SignInComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
write tests

Reply:
Removed/replaced this component with `HomeComponent` but will be including tests with the updates to the `AuthService`.